README.md

```diff
@@ -115,4 +115,8 @@ Run the test again and we should be passing the first of our two tests. Use the
 
 Define a method, `#introduction_with_language` that takes in two arguments, `name` and `language` and outputs the phrase: `"Hi, my name is #{name} and I am learning to program in #{language}."
```

Comment:
Looks like there is another issue on this page that might affect the rendering of the blurb below.  The inline code snippet above is missing a closing uptick.  Could you add that to this PR as well?